tests/unit/test_mxnet.py

```diff
@@ -66,6 +70,7 @@ def sagemaker_session():
     session.sagemaker_client.describe_training_job = Mock(return_value=describe)
     session.sagemaker_client.describe_endpoint = Mock(return_value=ENDPOINT_DESC)
     session.sagemaker_client.describe_endpoint_config = Mock(return_value=ENDPOINT_CONFIG_DESC)
+    session.sagemaker_client.list_tags = Mock(return_value=LIST_TAGS_RESULT)
```

Comment:
again, there is no test that the tags actually get added to the estimator. adding an assert in test_attach would cover this. since we are testing parent class behavior, we don't need to add this for every framework, but it would be nice to test it in one of them.